core-js/src/main/javascript/cdf/dashboard/Dashboard.js

```diff
@@ -35,7 +35,11 @@ define([
      * @extends {@link http://dean.edwards.name/weblog/2006/03/base/|Base}
      * @extends {@link http://backbonejs.org/#Events|Backbone.Events}
      * @classdesc Base abstract dashboard class.
-     * @param {Object} options Object that can contain the context, storage and view.
+     * @param {Object} [options] Object that can contain the context, storage and view.
+     * @param {Object} [options.context] The context of the dashboard retrieved from the server.
```

Comment:
add links? {@link cdf.dashboard.Dashboard#context|context} (same below if you agree)